src/lib/sqlite/global.ts

```diff
@@ -17,3 +20,9 @@ export const sqlitergExecutorStore = derived(
   }
 )
 export const localSnippetsStore = await createLocalSnippetsStore(executor, globalStateStore)
+export const remoteSnippetsStore = await createRemoteSnippetStore(globalStateStore, sqlitergExecutorStore)
+await waitUntil(remoteSnippetsStore.isAvailable)
+
+export const dataStateStore = createSnippetsDataStateStore(localSnippetsStore, remoteSnippetsStore)
+export const dataManager = createSnippetsDataManager(localSnippetsStore, remoteSnippetsStore, dataStateStore, 5_000)
+dataManager.start()
```

Comment:
This is having error because Drizzle underlies `remoteSnippetsStore` is parsing the returned data incorrectly (see https://github.com/proofrock/sqliterg/issues/5).